trillian/integration/hammer.go

```diff
@@ -499,7 +499,7 @@ func (s *hammerState) addChainInvalid(ctx context.Context) error {
 	return nil
 }
 
-// chooseCertToAdd determines whether to add a new or pre-existing cert.
+// chooseCertToAdd determines whether to add a new or preexisting cert.
```

Comment:
I think [`pre-existing`](https://dictionary.cambridge.org/dictionary/english/pre-existing) is perfectly cromulent :)